packages/react-charting/src/components/GaugeChart/GaugeChart.types.ts

```diff
@@ -24,6 +24,11 @@ export interface IGaugeChartSegment {
    */
   color?: string;
 
+  /**
+   * Gradient color of the segment
+   */
+  gradient?: string[];
```

Comment:
why do you need this property. I don't see this property in donut chart

Reply:
NB: take Tuple instead of array

Reply:
Resolved. All gradient properties are now defined as `gradient?: [string, string];`